pkg/kapp/clusterapply/applying_changes.go

```diff
@@ -85,25 +87,25 @@ func (c *ApplyingChanges) Apply(allChanges []*ctldgraph.Change) ([]WaitingChange
 			result := <-applyCh
 
 			c.ui.Notify(result.DescMsgs)
-
 			if result.Err != nil {
 				lastErr = result.Err
-				if result.Retryable {
-					continue
+				if !result.Retryable {
+					unsuccessfulChangeDesc = append(unsuccessfulChangeDesc, result.Err.Error())
```

Comment:
If a resource fails, then all the dependent resource in it in the change graph, what will happen to them ? Also, if a resource depends on two resources, one of them passes and other fails, then what is going to happen ?

Reply:
If a resource is blocked on another resource which has failed, then it won't get applied.